src/EnergyPlus/InputProcessing/IdfParser.cc

```diff
@@ -201,8 +201,18 @@ json IdfParser::parse_idf( std::string const & idf, size_t & index, bool & succe
 			auto const parsed_obj_name = parse_string( idf, index, success );
 			auto const obj_name = normalizeObjectType( parsed_obj_name );
 			if ( obj_name.empty() ) {
-				warnings_.emplace_back( "Line: " + std::to_string( cur_line_num ) + " Index: " + std::to_string( index_into_cur_line ) +
-										" - \"" + parsed_obj_name + "\" is not a valid Object Type. Attempting to skip object." );
+				errors_.emplace_back( "Line: " + std::to_string( cur_line_num ) + " Index: " + std::to_string( index_into_cur_line ) +
```

Comment:
@mbadams5 Thanks for the changes. For this case, it now stops after this without reporting any other input error and without the version mismatch warning. So, maybe this was better as it was before or you still report Severe but continue processing.